example/.gitignore

```diff
@@ -73,3 +73,7 @@ buck-out/
 
 # testing
 /coverage
+
+# Expo
+android/
+ios/
```

Comment:
Since the native directories are not in the repo, could you add `expo prebuild` as a step when the dependencies are installed? https://docs.npmjs.com/cli/v10/using-npm/scripts#life-cycle-scripts

Reply:
I guess, the better step is `postinstall`